lib/matobjplist.gi

```diff
@@ -1286,3 +1286,72 @@ InstallMethod( NewCompanionMatrix,
     return ll;
   end );
 
+############################################################################
+# Elementary matrix operations
+############################################################################
+
+############################################################################
+##
+#M  MultMatrixRow( <M>, <row>, <scalar> )
+##
+InstallMethod( MultMatrixRow, "for a mutable IsRowListMatrix, one row number, and a scalar",
+  [ IsRowListMatrix and IsMutable, IsInt, IsObject ],
```

Comment:
This is the wrong file; it is intended for the implementation of `IsPlistMatrix` (see also PR #2973). These methods should go into `lib/matobj.gi`.

Reply:
Or perhaps add a new file `lib/matrobjrowlist.gi` (to get GAP to actually read it, edit `lib/read5.g` around line 110 to add a suitable `Read` statement).